features/Dockerfile

```diff
@@ -0,0 +1,74 @@
+# syntax = docker/dockerfile:1.5
+# Used only for running tests using tox, see ../tox.ini
+ARG PG_MAJOR
+ARG PGHOME=/home/postgres
+ARG PGDATA=$PGHOME/data
+ARG LC_ALL=C.UTF-8
+ARG LANG=C.UTF-8
```

Comment:
Removed DATA.
The lang is useful incase `click` is called as it barfs without it set. Not sure if that is something we should be concerned here with but it might come up.

Reply:
ARG is only used in Dockerfile when referenced during the build. It is not set as env inside the container